app/components/App.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { createBootstrapComponent } from 'react-bootstrap/esm/ThemeProvider';
 import Wrapper from './Wrapper.jsx';
 
+// Renders the WRAPPER component
```

Comment:
why is this in caps?

Reply:
thought it would make it easy to read/scan quickly - you think it should be lowercase?